### ISSUE TITLE
installing 3.4.0 with python3 fails on linux depending on locale

### ISSUE THREAD

**Issue by benoit-monin** [getpelican/pelican#1409]
## Summary

Trying to install pelican 3.4.0 with python 3 on a linux box fails if the locale is not UTF-8. This seems to be caused by the pathname of some files containing non-ascii characters (e.g. décembre, février).

### Setup a virtualenv with python3
```
virtualenv -p python3 virtualenv_python3
cd virtualenv_python3
source bin/activate
```
### Set a non utf-8 locale
```
export LANG=C
```
or
```
export LANG=fr_FR
```
### Try to install pelican with pip
```
pip install pelican
```

## Results

It fails with the following traceback:
```
  File "<string>", line 1, in <module>
  File "/home/user/virtualenv_python3/build/pelican/setup.py", line 48, in <module>
    test_suite='pelican.tests',
  File "/usr/lib64/python3.3/distutils/core.py", line 148, in setup
    dist.run_commands()
  File "/usr/lib64/python3.3/distutils/dist.py", line 930, in run_commands
    self.run_command(cmd)
  File "/usr/lib64/python3.3/distutils/dist.py", line 949, in run_command
    cmd_obj.run()
  File "/home/user/virtualenv_python3/lib/python3.3/site-packages/setuptools/command/install.py", line 53, in run
    return _install.run(self)
  File "/usr/lib64/python3.3/distutils/command/install.py", line 574, in run
    self.run_command('build')
  File "/usr/lib64/python3.3/distutils/cmd.py", line 313, in run_command
    self.distribution.run_command(command)
  File "/usr/lib64/python3.3/distutils/dist.py", line 949, in run_command
    cmd_obj.run()
  File "/usr/lib64/python3.3/distutils/command/build.py", line 126, in run
    self.run_command(cmd_name)
  File "/usr/lib64/python3.3/distutils/cmd.py", line 313, in run_command
    self.distribution.run_command(command)
  File "/usr/lib64/python3.3/distutils/dist.py", line 949, in run_command
    cmd_obj.run()
  File "/home/user/virtualenv_python3/lib/python3.3/site-packages/setuptools/command/build_py.py", line 90, in run
    self.build_package_data()
  File "/home/user/virtualenv_python3/lib/python3.3/site-packages/setuptools/command/build_py.py", line 148, in build_package_data
    self.mkpath(os.path.dirname(target))
  File "/usr/lib64/python3.3/distutils/cmd.py", line 339, in mkpath
    dir_util.mkpath(name, mode, dry_run=self.dry_run)
  File "/usr/lib64/python3.3/distutils/dir_util.py", line 42, in mkpath
    if os.path.isdir(name) or name == '':
  File "/home/user/virtualenv_python3/lib64/python3.3/genericpath.py", line 41, in isdir
    st = os.stat(s)
UnicodeEncodeError: 'ascii' codec can't encode character '\u0301' in position 58: ordinal not in range(128)
```

The complete pip.log is [here](https://gist.github.com/benoit-monin/52818f2fe39b718753d2).

**Comment by ionelmc** [comment]
@justinmayer the simplest solution is to stop packaging the tests

**Comment by MinchinWeb** [comment]
This is a duplicate of #1545

**Comment by justinmayer** [comment]
Presumably addressed via #1778, which will be included in the next release.
